<?xml version="1.0" encoding="UTF-8"?>
<testsuite errors="1" failures="0" file="AprovePrivilege_tests.py" name="AprovePrivilege_tests.AprovePrivilege_test-20191030165654" skipped="0" tests="1" time="0.004" timestamp="2019-10-30T16:56:54">
	<testcase classname="AprovePrivilege_tests.AprovePrivilege_test" file="src/unittest/python/AprovePrivilege_tests.py" line="13" name="test_aprove" time="0.004" timestamp="2019-10-30T16:56:54">
		<error message="__init__() missing 1 required positional argument: 'user'" type="TypeError">
<![CDATA[Traceback (most recent call last):
  File "/home/joseluis/Documentos/ITESO/DSO/decorator-practice/python/src/unittest/python/AprovePrivilege_tests.py", line 11, in setUp
    self.priv = AprovePrivilege()
TypeError: __init__() missing 1 required positional argument: 'user'
]]>		</error>
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
</testsuite>
